addons/mail/models/mail_channel.py

```diff
@@ -1120,7 +1120,8 @@ def _execute_command_help(self, **kwargs):
         else:
             all_channel_partners = self.env['mail.channel.partner'].with_context(active_test=False)
             channel_partners = all_channel_partners.search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)])
-            msg = _("You are in a private conversation with <b>@%s</b>.", channel_partners[0].partner_id.name if channel_partners else _('Anonymous'))
+            #Show all people in a private conversation
+            msg = _("You are in a private conversation with <b>@%s</b>.", _(" @").join(partners.partner_id.name for partners in channel_partners) if channel_partners else _('Anonymous'))
```

Comment:
Note: there should be html escape around `partners.partner_id.name` as it is inserted into html code.

Reply:
While you are at it, it could be added around `self.name` line 1117 as well.